leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -573,6 +573,17 @@
       excluded.instanceField("android.view.Choreographer$FrameDisplayEventReceiver",
           "mMessageQueue").alwaysExclude();
     }
+  },
+
+  VIEWLOCATIONHOLDER_ROOT(SDK_INT >= 28) {
```

Comment:
- Should be `==` (P, not P+)
- Create a `P` constant for it at the top of the class

Reply:
P has a built-in constant, although using numbers is much better since no one remembers what the (dumb) letters correspond to.

Reply:
So the project is still compile SDK 28 :X. Was planning to to submit another PR to update deps and fix this with it, though Jake's also recommended using numbers on other projs for R8 (?) reasons

Reply:
The letter is only useful before an API level has been issued as way to refer to the next version. After that the raw number is self-documenting and more clear. All the docs and lint refer to the integer value and not the letter so it's just to stay consistent.